src/components/UserInfo.js

```diff
@@ -15,4 +16,7 @@ export default class UserInfo {
     this._nameEl.innerText = name;
     this._jobEl.innerText = job;
   }
+  setAvatar(link) {
```

Comment:
It's great to have a separate method for updating the avatar as it's controlled by the different form